examples/07_advanced/plot_beta_series.py

```diff
@@ -58,11 +58,14 @@
 .. include:: ../../../examples/masker_note.rst
 
 """
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    raise RuntimeError("This script needs the matplotlib library")
+
 # sphinx_gallery_thumbnail_number = -2
 
 # %%
-import matplotlib.pyplot as plt
-
 from nilearn import image, plotting
```

Comment:
here can't you also just switch the positions of importing nilearn.plotting and pyplot instead of raising the import error?

Reply:
If I do this then we get an isort error:


```
git diff

examples/07_advanced/plot_beta_series.py

────────────────────────┐
67: except ImportError: │
────────────────────────┘
│ 67 │                                                              │ 67 │
│ 68 │# %%                                                          │ 68 │# %%
│ 69 │from nilearn import image, plotting                           │ 69 │from nilearn import image, plotting
│    │                                                              │ 70 │import matplotlib.pyplot as plt
│ 70 │                                                              │ 71 │
│ 71 │# %%                                                          │ 72 │# %%
│ 72 │# Prepare data and analysis parameters                        │ 73 │# Prepare data and analysis parameters

(base) ╭─remi at remi-XPS-15-9570 in ~/github/nilearn/nilearn on output_examples✘✘✘ 24-01-22 - 9:48:39
╰─(base) ⠠⠵ pre-commit run -a
check python ast.........................................................Passed
check for case conflicts.................................................Passed
check json...............................................................Passed
check for merge conflicts................................................Passed
check toml...............................................................Passed
check yaml...............................................................Passed
fix end of files.........................................................Passed
mixed line ending........................................................Passed
trim trailing whitespace.................................................Passed
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing /home/remi/github/nilearn/nilearn/examples/07_advanced/plot_beta_series.py
```

Reply:
Could try to move this import closer to where it is used though

Reply:
No worries it's very minor so can just keep it as it is. I did't think about isort...